index.bs

```diff
@@ -1299,12 +1297,12 @@ To <dfn>deserialize key-value list</dfn> given a |serialized key-value list|:
       let |deserialized key| be |serialized key|.
 
    1. Otherwise let |deserialized key| be result of [=trying=] to [=deserialize local value=]
-      with |serialized key|.
+      given |realm| and |serialized key|.
 
    1. Let |serialized value| be |serialized key-value|[1].
 
-   1. Let |deserialized value| be result of [=trying=] to [=deserialize local value=] with
-      |serialized value|.
+   1. Let |deserialized value| be result of [=trying=] to [=deserialize local value=]
+      given |realm| and |serialized value|.
```

Comment:
I regularly get confused here, but I think the convention is to use `given` when defining an algorithm and `with` when calling it. So if we could update all the callers to use `with |realm| and |serialized value|` (or equivalent) that would help with consistency.